target/linux/ath79/image/generic.mk

```diff
@@ -3237,6 +3237,17 @@ define Device/xiaomi_mi-router-4q
 endef
 TARGET_DEVICES += xiaomi_mi-router-4q
 
+define Device/ditel_dt-acu100
+  SOC := qca9531
+  DEVICE_VENDOR := Ditel
+  DEVICE_MODEL := DT-ACU100
+  DEVICE_PACKAGES := -kmod-ath9k -uboot-envtools kmod-usb2
```

Comment:
Isn't `kmod-ath9k` necessary for the wmac on QCA9531?

Reply:
Okay, i will change to "DEVICE_PACKAGES := kmod-usb2"